apps/site/src/app/apply/sections/Form/ProfileInformation.tsx

```diff
@@ -0,0 +1,67 @@
+import RequiredAsterisk from "../Components/RequiredAsterisk";
+import styles from "./Form.module.scss";
+
+export default function ProfileInformation() {
+	return (
+		<div className="flex flex-col gap-5 w-11/12">
+			<p className="text-4xl m-0 font-bold text-center max-[700px]:text-3xl">
+				Profile Information
+			</p>
+
+			<div className="flex gap-5 w-full max-[1000px]:flex-col max-[1000px]:items-center">
```

Comment:
Perhaps it could be useful to make these textboxes components. They look the same in structure.